docs/troubleshooting-guide/sandboxing.md

```diff
@@ -0,0 +1,76 @@
+# Sandboxing
+
+[Sandboxing data][sandboxing-your-data] gives some people access to only a subset of the data. (The term comes from the practice of putting children in a sandbox to play safely.) To implement sandboxing, Metabase runs a query that filters rows and/or selects a subset of columns from a table; the user's query then runs on the initial query's result.
+
+Several databases did not support [common table expressions][cte] when sandboxing was added to Metabase, so we implemented it using subqueries. Suppose you use the Notebook Editor to create a query like:
+
+```
+select * from orders
+where price > 100.00
```

Comment:
We should be consistent with the SQL formatting here so it matches the format below.

Reply:
capitalized the keywords but didn't newline/indent - looks silly with just `*` on a line by itself